modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java

```diff
@@ -1037,8 +1119,105 @@ public void testObjectFieldSettings() throws Exception {
 		_assertObjectFieldSetting(
 			"showCounter", objectField.getObjectFieldId(), "false");
 
-		_objectDefinitionLocalService.deleteObjectDefinition(
+		// Default value
```

Comment:
@devpedrotavares can you change this comment to `// Business type picklist` to follow the same existing pattern? 

And I believe we should place it before the `// Business type text`